fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -720,7 +739,7 @@ public int searchCount(String sqlSelectCount) throws FHIRPersistenceDataAccessEx
 
     @Override
     public List<String> searchStringValues(SqlQueryData queryData)
-            throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException {
+        throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException {
```

Comment:
bad formatting change.  the double-indent signifies that its a continuation of the previous line so that it doesn't look like the first line of the method